test/cli/config-cwd-case-insensitive/_config.js

```diff
@@ -0,0 +1,13 @@
+var os = require( 'os' );
+
+function toggleCase ( s ) {
+  return ( s == s.toLowerCase() ) ? s.toUpperCase() : s.toLowerCase();
+}
+
+module.exports = {
+  skip: os.platform() !== 'win32',
```

Comment:
Test shouldn't be skipped because we need to make sure that your fix won't break another OS.